utils/similarity/__main__.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import sys
+import traceback
+import operator
+
+from .info     import info
+from .test     import test
+from .train    import train
+from .plot     import plot
+
+logging.basicConfig()
+logger = logging.getLogger("Slither-simil")
```

Comment:
Consider not mixing custom logger and `print` usage. Al the print can probably be changed to `logger.info()`

Reply:
It should be fixed now.